src/main/java/edu/greenblitz/tobyDetermined/subsystems/telescopicArm/Elbow.java

```diff
@@ -0,0 +1,94 @@
+package edu.greenblitz.tobyDetermined.subsystems.telescopicArm;
+
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkMaxLowLevel;
+import edu.greenblitz.tobyDetermined.RobotMap;
+import edu.greenblitz.tobyDetermined.subsystems.GBSubsystem;
+import edu.greenblitz.utils.motors.GBSparkMax;
+import edu.wpi.first.math.controller.ProfiledPIDController;
+import edu.wpi.first.math.trajectory.TrapezoidProfile;
+
+
+public class Elbow extends GBSubsystem {
+
+
+    private static Elbow instance;
+    public ElbowState state = ElbowState.IN_ROBOT;
+    public GBSparkMax motor;
+
+    public static Elbow getInstance(){
+        if(instance == null){
+            instance = new Elbow();
+        }
+        return instance;
+    }
+
+    private Elbow(){
+        motor = new GBSparkMax(RobotMap.telescopicArm.elbow.MOTOR_ID, CANSparkMaxLowLevel.MotorType.kBrushless);
+
+        motor.config(new GBSparkMax.SparkMaxConfObject()
+                .withPID(RobotMap.telescopicArm.elbow.PID)
+                .withPositionConversionFactor(RobotMap.General.Motors.SPARKMAX_TICKS_PER_RADIAN)
+        );
+        motor.getEncoder().setPosition(0); //resetPosition
+        motor.setSoftLimit(CANSparkMax.SoftLimitDirection.kReverse, RobotMap.telescopicArm.elbow.BACKWARD_ANGLE_LIMIT);
+        motor.setSoftLimit(CANSparkMax.SoftLimitDirection.kForward, RobotMap.telescopicArm.elbow.FORWARD_ANGLE_LIMIT);
+    }
+
+    public void setAngle(double angleInRads) {
```

Comment:
the name of the function sould reflect the fact that you cant just set the angle once but have to call it periodicly usiing the execute